Assets/NuGet/Editor/RepositoryType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NugetForUnity
```

Comment:
This file needs to be included in the CreateDLL solution - right now build.ps1 fails as it's missing RepositoryType.